packages/agoric-cli/src/commands/inter.js

```diff
@@ -153,7 +299,7 @@ For example:
 }
 `,
     )
-    .option('--manager [number]', 'Vault Manager', Number, 0)
+    .option('--manager <number>', 'Vault Manager', Number, 0)
```

Comment:
I forget, what difference does this make? I thought the brackets signaled "optional" which is correct here.

Reply:
The brackets means that the `--manager` can be given without the `number`. I suppose it's correct in that it would default to 0, but that seems kinda weird.